apps/web/src/pages/api/v1/admin/get-claims.ts

```diff
@@ -0,0 +1,26 @@
+import { BadRequest } from 'http-errors'
+import { NextApiResponse } from 'next'
+
+import configureAdmin from '@/clients/firebase-admin-client'
+import createHandler, { NextApiRequestApp } from '@/middleware'
+import adminMiddleware from '@/middleware/admin-middleware'
+import authMiddleware from '@/middleware/auth-middleware'
+import userMiddleware from '@/middleware/user-middleware'
+
+const handler = createHandler()
+handler.use(authMiddleware()).use(userMiddleware()).use(adminMiddleware())
+
+// Get claims for logged in user
+handler.get(async (request: NextApiRequestApp, response: NextApiResponse) => {
+  if (!request.user.externalId) {
+    throw new BadRequest('No external ID provided')
+  }
+  const admin = configureAdmin()
+  const firebaseUser = await admin.auth().getUser(request.user.externalId)
+  const claims = firebaseUser.customClaims
+    ? Object.keys(firebaseUser.customClaims)
+    : []
+  response.status(200).json({ roles: claims })
```

Comment:
It's a little confusing that `get-claims` returns `roles`. I think we should either return the claims as they are returned to us by firebase or rename this to `get-roles`.